packages/odyssey-storybook/src/components/Field/Field.mdx

```diff
@@ -1,43 +1,150 @@
 
-Please refer to individual components for complete documentation. These examples are provided for quick illustration.
+When assembling Forms, it's unnecessary to utilize Field, as it's built into Odyssey's Inputs already.
 
-### Label
+If you are building a new or bespoke Input component, please utilize Field to ensure consistent behavior with existing components.
+
+## Content Guidelines
+
+### Labels
+
+A well-written label describes the purpose of the Field for the user.
+
+Use concise, descriptive labels (a word or two) to help users quickly scan your form.
+
+Finally, please use sentence casing for your labels. Sentence case is easier to read and will help reduce error rates and speed up form completion time.
+
+### Hints
+
+When writing hints, only provide information that assists with the immediate task.
+
+For example, "The expected format is `YYYY-MM-DD`."
+
+Hints like, "This information will be used to do x, y, and z." may be informative without assisting the user in completing their form.
+
+If additional disclosures or warnings are required, an [Infobox](?path=/docs/components-infobox--info#infobox) may be more appropriate.
 
-Labels caption pieces of UI, typically form fields. They are required for all inputs. When related to sets of Radios or Checkboxes, they may also be referred to as "legends".
+Remember: a good label may eliminate the need for a hint.
 
-Keep labels to a word or two so users can quickly scan the form. Always use sentence casing.
+### Errors
 
-**STORY MISSING**
+Error messaging should explain: (1) what has gone wrong and (2) how the user can fix the issue.
 
-### Field hint
+"This value is invalid." is less helpful than "This field requires a complete email address."
 
-Use field hints to provide context, formatting help, or other guidelines to the user. Do not use the placeholder attribute for these purposes.
+If the user cannot fix the error or if the error extends beyond the Field, it should instead be surfaced at the Field Group, Form, or page level.
 
-Limit hints to a short sentence. If additional context is required, it should be provided outside the Field.
+## Accessibility
 
-**STORY MISSING**
+Field does its best to correctly associate attributes and place `aria-` tags where necessary. For input-specific accessibility concerns, please refer to the relevant component's documentation.
 
-### Error messages
+Your error messages should include an "Error:" label, whether visible or visually hidden.
 
-Invalid fields should include an inline message to explain the error.
+## Globalization
 
-If possible, describe how the error may be resolved.
+To support ease of localization, Field's `optionalLabel` string has no default and must be supplied by the user.
 
-**STORY MISSING**
+Similarly, an "Error:" label is not automatically inserted into Field error messages.
```

Comment:
the globalization notes here feel superfluous - I'm not a fan of documenting what our components do not do. Perhaps reword this to simply mention that all user facings strings, be it labels, hints, placeholders, or error text are the responsibility of the consuming developer to translate? This may already be apparent for some developers.

Reply:
Designers, PMs and others may not know what the components do. I don't think being explicit about behavior is a bad thing.